package.json

```diff
@@ -19,7 +19,8 @@
 		"npm": ">=6.9.0"
 	},
 	"config": {
-		"GUTENBERG_PHASE": 2
+		"GUTENBERG_PHASE": 2,
+		"COMPONENT_SYSTEM_PHASE": 0
```

Comment:
Why is this flag used for? In't it redundunt with the "context" component provider?

Reply:
We want to merge this PR with all the new code removed (100kb as of today according to the CI job) when running `npm run build`. This way we can integrate a few components, remove code duplication wherever possible and enable this flag (or remove completely) afterwards.

Reply:
What's the point of merging components and not really use them in the plugin?

Reply:
I can also confirm now that Tree Shaking works like a charm:
<img width="620" alt="Screen Shot 2020-12-10 at 09 18 48" src="https://user-images.githubusercontent.com/699132/101740332-f8fafa80-3ac8-11eb-8f67-0959b6c86d05.png">

🎉